test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -178,7 +200,20 @@ def test_smmodelparallel_mnist_multigpu_multinode(ecr_image, instance_type, py_v
                     "custom_mpi_options": "-verbose --mca orte_base_help_aggregate 0 -x SMDEBUG_LOG_LEVEL=error -x OMPI_MCA_btl_vader_single_copy_mechanism=none",
                 },
             },
-        )
+        }
+        try:
+            pytorch = PyTorch(
+                image_uri=ecr_image,
+                sagemaker_session=sagemaker_session,
+                **estimator_parameter
+                )
+        except Exception as e:
+            if type(e) == sagemaker.exceptions.UnexpectedStatusException and "Capacity Error" in str(e):
```

Comment:
did you verify that it works? If there is no capacity in us-west-2, does it run in us-east-1?

Reply:
yes